src/bundle.rs

```diff
@@ -25,6 +25,20 @@ pub unsafe trait DynamicBundle {
         None
     }
 
+    /// Checks if the Bundle contains the given `T`:
+    ///
+    /// ```
+    /// # use hecs::DynamicBundle;
+    ///
+    /// let my_bundle = (0i32, 10.0f32);
+    /// assert!(my_bundle.has::<i32>());
+    /// assert!(my_bundle.has::<f32>());
+    /// assert!(!my_bundle.has::<usize>());
+    /// ```
+    fn has<T: Component>(&self) -> bool {
```

Comment:
`EntityBuilder`-related bundles could probably have a more efficient override for this.

Reply:
Want me to take a crack at implementing that, or are you asking me to not provide a default implementation?

Reply:
Now that you mention it, implementing this directly for each type rather than with this default might not be a bad idea, since it lets us skip the sorting in the type ID accessors.